js/alcatraz-theme.js

```diff
@@ -156,11 +156,44 @@
 		});
 	}
 
+	function aczSidebarNav() {
```

Comment:
This is a great effort. There are definitely some opportunities for refactoring this into a single function that we can use for both the sub navs in the primary menu and the section nav, but it's going to be complicated. I wouldn't worry about the JS side of this thing for now, just get the PHP as solid as you can and then we can revisit the JS later.